main.py

```
@@ -0,0 +1,375 @@
+    # Parse ISO 8601 datetime string to datetime object
+    v = value.strip()
+    if v.endswith("Z"):
+        v = v[:-1] + "+00:00"
+    parsed = datetime.fromisoformat(v)
+    if parsed.tzinfo is None:
+        return parsed.replace(tzinfo=timezone.utc)
+    return parsed
+
+
+@app.get("/calendar/event/{event_id}/ics", description="Create ICS file for an event", tags=["calendar"])
+async def download_event_ics(event_id: str, client: httpx.AsyncClient = Depends(_get_cms_client)):
```

Ruff (0.14.10)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)